flink-core/pom.xml

```diff
@@ -63,6 +63,19 @@ under the License.
 			<artifactId>guava</artifactId>
 			<version>${guava.version}</version>
 		</dependency>
+
+        <dependency>
```

Comment:
Seems to be ASL 2.0 (https://github.com/addthis/stream-lib/blob/master/LICENSE.txt) --> good